.golangci.yml

```diff
@@ -14,17 +14,14 @@ linters-settings:
 linters:
   disable-all: true
   enable:
-    - deadcode
     - gofmt
     - gosimple
     - govet
     - ineffassign
     - exportloopref
-    - structcheck
     - staticcheck
     - unconvert
     - unused
-    - varcheck
```

Comment:
These 3 rules have been deprecated after v1.49.0. Now they are handled by the `unused` rule.